data/strings/types_strings.txt

```diff
@@ -17279,38 +17291,14 @@
     zh-Hant = 理髮師
 
   [type.shop.hardware]
-    en = Hardware Store
-    ar = متجر عدد وأدوات
-    cs = Železářství
-    da = Isenkræmmer
-    de = Eisenwarengeschäft
```

Comment:
Why deleted? Some translations here are different from `doityourself`.
Don't delete translations manually - add a ref, then run the generate_localizations script ad it will remove duplicating ones automatically while preserving unique ones.